src/main.rs

```diff
@@ -41,21 +41,21 @@ const USAGE: &str = r#"
 Spellcheck all your doc comments
 
 Usage:
-    cargo-spellcheck [(-v...|-q)] check [--cfg=<cfg>] [--code=<code>] [--skip-readme] [--checkers=<checkers>] [[--recursive] <paths>... ]
-    cargo-spellcheck [(-v...|-q)] fix [--cfg=<cfg>] [--code=<code>] [--skip-readme] [--checkers=<checkers>] [[--recursive] <paths>... ]
+    cargo-spellcheck [(-v...|-q)] check [--cfg=<cfg>] [--code=<code>] [--skip-readme] [--reachable-link] [--checkers=<checkers>] [[--recursive] <paths>... ]
+    cargo-spellcheck [(-v...|-q)] fix [--cfg=<cfg>] [--code=<code>] [--skip-readme] [--reachable-link] [--checkers=<checkers>] [[--recursive] <paths>... ]
     cargo-spellcheck [(-v...|-q)] config (--user|--stdout|--cfg=<cfg>) [--force]
-    cargo-spellcheck [(-v...|-q)] [--cfg=<cfg>] [--fix] [--code=<code>] [--skip-readme] [--checkers=<checkers>] [[--recursive] <paths>... ]
+    cargo-spellcheck [(-v...|-q)] [--cfg=<cfg>] [--fix] [--code=<code>] [--skip-readme] [--reachable-link] [--checkers=<checkers>] [[--recursive] <paths>... ]
     cargo-spellcheck --help
     cargo-spellcheck --version
 
 Options:
   -h --help                 Show this screen.
   --version                 Print the version and exit.
 
-  --fix                     Interactively apply spelling and grammer fixes, synonym to `fix` sub-command.
+  --fix                     Interactively apply spelling and grammar fixes, synonym to `fix` sub-command.
```

Comment:
Maybe we should eventually include all literals to be checked with the spell checker...

Reply:
makes sense, I just have autocorrector in my IDE. I also use https://languagetoolplus.com/?pk_campaign=addon2-errorcard-logo on the browser :)

Reply:
If I could limit it to certain websites, that would be an option, but it seems a bit too invasive to allow it sending all textual content of my browser to a server out of my reach. Thanks for the suggestion! I'll check if there are sufficient tuning knobs for my purpose.